src/middlewares/requests.middleware.js

```diff
@@ -0,0 +1,24 @@
+import models from '../db/models';
+
+const { Request } = models;
+
+const requestChecks = {
+  async validateRequests(req, res, next) {
```

Comment:
I think this should be handled by a validation middleware. Also, don't you think a 422 status code is more appropriate?

Reply:
Alright. Let me implement your requested changes. I will have to break out only the `validate ID` part of it shaa, because checking if a user exists should be in the middleware folder. Thanks again for your suggestions